dev/releases/release_notes.py

```diff
@@ -316,6 +345,40 @@ def changes_overview(
         # finally copy over this new file to changelog.md
         os.rename(newfile, finalfile)
 
+def split_pr_into_changelog(prs: List):
+    childprlist = []
+    toremovelist = []
+    for pr in prs:
+        if has_label(pr, 'release notes: use body'):
+            mdstring = body_to_release_notes(pr).strip()
+            mdlines = mdstring.split('\n')
+            pattern = r'\{.*\}$'
+            for line in mdlines:
+                mans = re.search(pattern, line)
+                if not mans:
+                    # maybe actually raise an error instead of just continue
```

Comment:
Is this a TODO comment? Then
```suggestion
                    # TODO: maybe actually raise an error instead of just continue
```